modules/google-analytics/classes/wp-google-analytics-universal.php

```diff
@@ -49,6 +49,13 @@ public function wp_head() {
 			return;
 		}
 
+		if ( class_exists( 'Jetpack_AMP_Support' )
+			&& Jetpack_AMP_Support::is_amp_request()
+		) {
+			add_filter( 'amp_analytics_entries', 'Jetpack_Google_Analytics::amp_analytics_entries' );
```

Comment:
This will only work for Transitional and Standard modes. We need an implementation for Reader mode too. See: https://amp-wp.org/documentation/playbooks/analytics/

Reply:
Great catch @MattGeri, thanks.